documentation/c01-whatis.sil

```diff
@@ -82,11 +82,105 @@ SILE does not benefit from the large ecosystem and community that has grown up a
 In that sense, TeX will remain ahead of SILE for some time to come.
 But in terms of \em{core capabilities}, SILE aims at being at least equivalent to TeX.
 
+\section{SILE versus Typst and other new alternatives}
+
+Almost exactly one decade after SILE development started, a new competitor to LaTeX entered the scene with a bang:
+\footnote{Days of front-page coverage on Hacker News, Lobsters, and the like propelled it past 10k GitHub stars in the first 6 days of being public, a milestone SILE is still very far from reaching.}
+Typst.
+There is no shame in stating that Typst is a great piece of software and a worthy competitor to SILE as well as well as LaTeX.
```

Comment:
Why do we want "competitors" (and even to compete at all) ? Using the right tools for the right needs is not only about _competition_, or being "worthy" of anything in general.